go/libkb/loaduser.go

```diff
@@ -304,7 +304,9 @@ func LoadUserFromServer(g *GlobalContext, uid keybase1.UID, body *jsonw.Wrapper)
 		if err != nil {
 			return nil, err
 		}
-		body = res.Body.AtKey("them")
+		if body, err = res.Body.AtKey("them").ToArray(); err != nil {
```

Comment:
I'm confused as to how in `resolve.go`, the `AtKey("them")` error passes through to KBFS, but here it gets automagically converted to `SCNotFound`.

```
92155 2016-08-03T10:31:17.720242 ▶ [DEBU keybase util.go:387] 1ab0 - resolveURLViaServerLookup(input = "uid:a23de2bbc16771fdfb6ff32fc1d99f19") -> ERROR: <root>.them: no such key: them
92156 2016-08-03T10:31:17.720288 ▶ [DEBU keybase util.go:387] 1ab1 - Resolving full expression "uid:a23de2bbc16771fdfb6ff32fc1d99f19" -> ERROR: <root>.them: no such key: them
92157 2016-08-03T10:31:17.720320 ▶ [DEBU keybase util.go:387] 1ab2 - IdentifyHandler.Resolve2(uid:a23de2bbc16771fdfb6ff32fc1d99f19) -> ERROR: <root>.them: no such key: them
```